cmd/zitadel/main.go

```diff
@@ -89,6 +92,10 @@ const (
 )
 
 func main() {
+	err := sentry.Init(sentry.ClientOptions{})
+	if err != nil {
+		log.Fatalf("sentry.Init: %s", err)
+	}
```

Comment:
IMHO we should only start sentry by setting a flag or at least have a flag to disable it

Reply:
Should we not just rely on the input values?
E.g. when no config is given we don't load this?

Reply:
sound reasonable, a shell variable should fix that

Reply:
why exactly a shell variable?

Reply:
thats what ops people only care about :D
call it "variable" 

on it